unixfs/io/bufdagreader.go

```diff
@@ -0,0 +1,41 @@
+package io
+
+import (
+	"bytes"
+	"context"
+	"io"
+)
+
+type bufDagReader struct {
```

Comment:
Yeah, the interface of DagReader supersedes RSC and bufDagReader is nop-closers on bytes.